packages/terra-date-time-picker/tests/wdio/date-time-picker-spec.js

```diff
@@ -134,7 +134,7 @@ describe('DateTimePicker', () => {
       browser.url('/#/raw/tests/terra-date-time-picker/date-time-picker/date-time-picker-on-change');
       browser.execute(() => {
         // Removes the blinking cursor to prevent screenshot mismatches.
-        document.querySelector('input[name="terra-time-minute-input"]').style.caretColor = 'transparent';
```

Comment:
I don't think we want to include the changes from 81a20107e86bf56fd701979316133da880cd3f15 (this change and similar) because it reverts the changes from #1673